ROADMAP.md

```diff
@@ -8,85 +8,102 @@ Each weekly release will include necessary tasks that lead to the completion of
+### Amazon Quicksight
+Issue: [#10990](https://github.com/hashicorp/terraform-provider-aws/issues/10990)
 
-Support for Amazon Timestream will include:
+_Amazon QuickSight is a scalable, serverless, embeddable, machine learning-powered business intelligence (BI) service built for the cloud. QuickSight lets you easily create and publish interactive BI dashboards that include Machine Learning-powered insights. QuickSight dashboards can be accessed from any device, and seamlessly embedded into your applications, portals, and websites._
+
+Support for Amazon Quicksight will include:
 
 New Resource(s):
+- aws_quicksight_data_source
+- aws_quicksight_group_membership
+- aws_quicksight_iam_policy_assignment
+- aws_quicksight_data_set
+- aws_quicksight_ingestion
+- aws_quicksight_template
+- aws_quicksight_dashboard
+- aws_quicksight_template_alias
+
 
-- aws_timestreamwrite_database
+### Amazon AppStream
+Issue: [#6058](https://github.com/hashicorp/terraform-provider-aws/issues/6508)
 
-### AWS AppConfig
+_Amazon AppStream 2.0 is a fully managed non-persistent desktop and application virtualization service that allows your users to securely access the data, applications, and resources they need, anywhere, anytime, from any supported device. With AppStream 2.0, you can easily scale your applications and desktops to any number of users across the globe without acquiring, provisioning, and operating hardware or infrastructure. AppStream 2.0 is built on AWS, so you benefit from a data center and network architecture designed for the most security-sensitive organizations. Each end user has a fluid and responsive experience because your applications run on virtual machines optimized for specific use cases and each streaming sessions automatically adjusts to network conditions._
 
-Issue: [#11973](https://github.com/hashicorp/terraform-provider-aws/issues/11973)
+Support for Amazon AppStream will include:
 
-_Use AWS AppConfig, a capability of AWS Systems Manager, to create, manage, and quickly deploy application configurations. You can use AWS AppConfig with applications hosted on Amazon Elastic Compute Cloud (Amazon EC2) instances, AWS Lambda, containers, mobile applications, or IoT devices._
+New Resource(s):
+- aws_appstream_stack
+- aws_appstream_fleet
+- aws_appstream_imagebuilder
 
-Support for AWS AppConfig will include:
+### Amazon Connect 
```

Comment:
@AdamTylerLynch Updates to Roadmap for connect; let me know if there are any changes you'd like me to make!